git-remote-gosh/src/blockchain/mod.rs

```diff
@@ -94,6 +113,13 @@ struct CallResult {
     out_msgs: Vec<String>,
 }
 
+#[derive(Deserialize, Debug)]
+struct FFCallResult {
```

Comment:
Please rename according to the method name

Reply:
^

Reply:
Also from dev blame says it was Vasiliy